tomls/omnibus-arbitrum-sepolia-octopus/perps/global.toml

```diff
@@ -0,0 +1,97 @@
+[setting.settlementReward]
```

Comment:
seems like a good time to swithc to use `[var.perps_settings]` and move to the main toml